### PR TITLE
Add Frenchtwinks

### PR DESCRIPTION
**Pull request by gimmeliina**

### CODE REVIEW

**Comment by Belleyy** [comment]
You need to update `SCRAPERS-LIST.md`

File: scrapers/Frenchtwinks.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1[@class="big_title2017"]
+      Details: 
+        selector: //div[@class="vid_txt_block2 vid_txt_block2017"]
+      Date:
+        selector: //time[@datetime]
+        postProcess:
+          - parseDate: 2021-01-02
```
Comment (bnkai) [inline scrapers/Frenchtwinks.yml:16]: typo
```suggestion
          - parseDate: 2006-01-02
```

File: scrapers/Frenchtwinks.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      Image: 
+        selector: //meta[@property="og:image"]/@content
+      Studio:
+        Name:
+          fixed: French Twinks
+      Movies:
+        Name:
```
Comment (bnkai) [inline scrapers/Frenchtwinks.yml:23]: leftover? needs to be removed

File: scrapers/Frenchtwinks.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  - action: scrapeXPath
+    url:
+      - french-twinks.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1[@class="big_title2017"]
+      Details: 
+        selector: //div[@class="vid_txt_block2 vid_txt_block2017"]
```
Comment (bnkai) [inline scrapers/Frenchtwinks.yml:12]: The selector includes the twitter/tumbler/reddit buttons
Either use 
```yaml
      Details: 
        selector: //div[@class="vid_txt_block2 vid_txt_block2017"]/article
```
or 
```yaml
      Details: 
        selector: //div[@class="vid_txt_block2 vid_txt_block2017"]/article//text()
        concat: "\n\n"
```

````

**Pull request merged**